### PR TITLE
Fix `createTheme`

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

- [x] Add missing `@mui/utils` package,
- [x] Move `styles` into `src` folder and ensure it's exported from there,
- [x] Remove unused `storybook-addon-material-ui5`, clean up `packages/config`, and ensure **Roboto** fonts are loaded,
- [x] Improve `createContentWidths` to use `breakpoints` when a user has provided custom `breakpoints` without providing `contentWidths`, and
- [x] Ensure `createContentwidths` is fully tested.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by nerdlet — APPROVED** [review]
@kilemensi Looks good 🚀 🚀

File: packages/commons-ui-core/src/styles/createTheme/createContentWidths.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    return contentWidths;
+  }
+  if (breakpoints) {
+    // Use https://mui.com/customization/breakpoints/#default-breakpoints
+    // as is for content widths i.e. body
+    const { values: breakpointValues, unit } = breakpoints;
+    const values = { ...breakpointValues };
+    // Assume values are sorted, remove the first value so that content width
+    // is 100% of the size.
+    delete values[Object.keys(breakpointValues)[0]];
```
Comment (KhadijaMahanga) [inline packages/commons-ui-core/src/styles/createTheme/createContentWidths.js:12]: @kilemensi by sorted, do you mean sorted alphabetically? or sorted per screensize? 

If `xs` is what we need to delete, why not perform delete by just using `xs` key

**Review by KhadijaMahanga — APPROVED** [review]
LG 🚀

**Review by gertie-sheshe — APPROVED** [review]
👍

File: packages/commons-ui-core/src/styles/createTheme/createContentWidths.js (hunk 1 above)
Comment (kilemensi) [reply packages/commons-ui-core/src/styles/createTheme/createContentWidths.js:12]: By screen size @KhadijaMahanga ... and MUI allows [completely](https://mui.com/material-ui/customization/breakpoints/#custom-breakpoints) customization of key names so we can't assume it'll be always `xs`.

**Pull request merged**